low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -0,0 +1,153 @@
+#!/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+###################################################################
+# post_install script configures ryslog, sspl service and required
+# site packages for SSPL. This is executed from sspl_setup script.
+###################################################################
+
+import os
+import errno
+import shutil
+import distutils.dir_util
+
+# using cortx package
+from cortx.sspl.bin.error import SetupError
+from cortx.sspl.bin.sspl_constants import (REPLACEMENT_NODE_ENV_VAR_FILE,
+                                           SSPL_BASE_DIR,
+                                           file_store_config_path,
+                                           sample_global_config,
+                                           PRODUCT_BASE_DIR)
+from cortx.sspl.lowlevel.framework import sspl_rabbitmq_reinit
+from cortx.utils.process import SimpleProcess
+from cortx.utils.service import Service
+from cortx.utils.conf_store import Conf
+from cortx.utils.conf_store.error import ConfError
```

Comment:
import order needs to be changed. first built-in modules, second third party, third sspl modules.

Reply:
Done.